### ISSUE TITLE
Unable to use en_AU or en_GB in a reliable manner?

### ISSUE THREAD

**Issue by abraunegg** [check-spelling/check-spelling#76]
According to the docs, to support Spanish, the following is needed:
![image](https://github.com/user-attachments/assets/98db607a-6d3a-491b-bcc4-da22bd9e7a39)

When this is augmented to configure the application to use en_AU and/or en_GB this is what I have configured:
![image](https://github.com/user-attachments/assets/355f020f-268d-4df3-b8b2-15a97d19f2fe)

This *should* be mapping to what is on the check-spelling github repository, similar to Spanish:
![image](https://github.com/user-attachments/assets/b03adf37-e0ec-421b-82e5-9f797dcb94d7)

However, despite this configuration to use en_AU, and confirming that words being used are in the AU dictonary (such as capitalisation) , the spell checker still says that 'capitalisation' is an unrecognised word.

#### Dictionary Entry
![image](https://github.com/user-attachments/assets/8ff2f275-d89f-45ac-b344-8076597a4254)

#### Unrecognised Word
![image](https://github.com/user-attachments/assets/7880e6fb-f3e7-4d2f-ad51-ebb59fe49372)

How can I reliably force the spell checker to use en_AU / en_GB for spelling or set this as a default?

**Comment by abraunegg** [comment]
Self resolved:

Need to modify 'master' to add the dictionary rather than a Pull Request

**Comment by jsoref** [comment]
My todo list does include warning people when a PR changes the configuration as those changes aren't applied until the changes are merged as you discovered.

In the interim, I've added an item to https://docs.check-spelling.dev/Known-Issues.html#changes-to-the-workflow-will-not-be-tested-in-a-pull-request-if-the-workflow-is-configured-to-use-codepullrequesttargetcode.

Sadly, as it happens, while the instructions worked for you, they don't work today for Spanish:
* #80 
I've just figured out what went wrong and hope to fix it tonight or tomorrow at the latest (along with adding testing).
